tests/test_formats.py

```diff
@@ -82,6 +87,8 @@ def test_date_format():
     t.set_format('time', ds.DateFormatter("%Y-%m-%d %H:%M:%S.%f"))
     assert isinstance(t['time'][0], float)
 
+    fmt = ds.DateFormatter("%Y-%m-%d").format_value
+    assert_equal(fmt(0), "1970-01-01")
```

Comment:
This yields me "1969-12-31" instead of "1970-01-01" due to time zones. Can we make the test so that it accepts either value instead? We want to make sure that tests don't fail on local machines